packages/engine.test/src/testView.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { Engine } from "@c11/engine.react";
+import { generateImage } from 'jsdom-screenshot';
+
+jest.useFakeTimers();
+
+export async function testView(name: string, { state, View, props }) {
+    test(name, async () => {
+        const root = document.createElement("div")
+        const title = document.createElement("h1")
+        title.innerHTML = name
+        
+        document.body.innerHTML = ""
+        document.body.appendChild(title)
+        document.body.appendChild(root)
+      
+        const engine = new Engine({
+          state: {
+              initial: state
+          },
+          view: {
+            element: <View
+              {...props}
+            />,
+            root
+          },
+        });
+      
+        jest.runAllTimers();
+      
+        const screenshot = await generateImage();
+        expect(screenshot).toMatchImageSnapshot();
```

Comment:
The developer should be allowed to pass a configuration object to customize their use case regarding testing the snapshot.
https://github.com/americanexpress/jest-image-snapshot

Reply:
`jest-image-snapshot` is 10 times slower that the `jest-screenshot` that we are using and it directly supports config file in root project
https://www.npmjs.com/package/jest-screenshot#configuring

Reply:
Sorry, I've linked the wrong package. About the config, the same point as above - should avoid exposing this to the project.

Reply:
for now it's good enough, we can register this as a future improvement